workflow/scripts/de_analysis.py

```
@@ -1,32 +1,34 @@
+from bioinfokit import visuz
 
 
 from snakemake.exceptions import WorkflowError
 
 from pydeseq2.dds import DeseqDataSet
 from pydeseq2.ds import DeseqStats
-from pydeseq2.utils import load_example_data
-
+from pydeseq2.default_inference import DefaultInference
 
 sys.stderr = sys.stdout = open(snakemake.log[0], "w")
```

Ruff

Use a context manager for opening files

(SIM115)